CITATION.cff

```diff
@@ -30,7 +30,11 @@ authors:
   family-names: Bauer
   affiliation: Princeton University
   orcid: https://orcid.org/0000-0002-4026-740X
-version: '1.2'
+- given-names: Kevin
+  family-names: McElwee
+  affiliation: '@Princeton-CDH'
```

Comment:
Please update affiliation to "Princeton University" for everyone who still works here (I guess just you and me!?) and remove affiliation for everyone else.